zeroex/order_validator.go

```diff
@@ -654,6 +685,108 @@ func (o *OrderValidator) isSupportedAssetData(assetData []byte) bool {
 	return true
 }
 
+// findChunkSize uses a limited-depth binary-search to find a chunk size that is less then the
+// max content length
+func (o *OrderValidator) findChunkSize(signedOrders []*SignedOrder) int64 {
+	// Determine the lowest and highest possible signedOrders limits to binary search in between
+	lo := int64(0)
+	hi := int64(len(signedOrders))
+	min := int64(1)
+
+	// Execute the binary search and hone in on an executable payload size of signedOrders
+	maxBinarySearchDepth := 7
+	depth := 0
+	for lo+1 < hi {
```

Comment:
Optionally, you could write the for loop this way:

```suggestion
	for depth := 0; lo+1 < hi && depth <= maxBinarySearchDepth; depth++ {
```